src/Jenssegers/Mongodb/Query/Builder.php

```diff
@@ -833,7 +848,7 @@ protected function performUpdate($query, array $options = [])
         }
 
         $wheres = $this->compileWheres();
-        $result = $this->collection->UpdateMany($wheres, $query, $options);
+        $result = $this->collection->UpdateMany($wheres, $query, array_merge($options, $this->session()));
```

Comment:
As per signature below:
```suggestion
        $result = $this->collection->UpdateMany($wheres, $query, $this->session($options));
```

There are a few more instances of this in the class.